__tests__/app.test.js

```diff
@@ -21,3 +27,38 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles", () => {
+  it("retrieves array of articles in decending order by date", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+
+        expect(articles).toHaveLength(13);
+
+        articles.forEach((article, i) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+
+        for (let i = 0; i < articles.length - 1; i++) {
+          if (
+            Date.parse(articles[i].created_at) <
+            Date.parse(articles[i + 1].created_at)
+          ) {
+            expect(false).toBe(true);
+          }
```

Comment:
This feels a bit hacky, you could have expected articles[i].created_at tobelessthan articles[i + 1].created_at

Reply:
or better yet, jest-sorted